interface/breeze-electron/src/components/Vitals/Vitals.jsx

```diff
@@ -81,11 +85,21 @@ export default class Vitals extends React.Component {
       : supportParams;
     let footer = (
         <div style={{position: 'relative' }}>
-          <div style={{paddingLeft: "20px"}}>Mode:</div>
+          <div style={{paddingLeft: "20px"}}>
+          <InputLabel id="label"> { this.state.mode } </InputLabel>
```

Comment:
use {mode} instead of {this.state.mode} :)

Reply:
thx! :)